apps/wallet/src/ui/app/staking/stake/StakingCard.tsx

```diff
@@ -115,16 +164,34 @@ function StakingCard() {
             setSendError(null);
             try {
                 const bigIntAmount = parseAmount(amount, coinDecimals);
-                // TODO: add unstake functionality
-                if (unstake) return;
-                const response = await dispatch(
-                    stakeTokens({
+                let response;
```

Comment:
Not a blocking concern, but I think this could be done a little cleaner with `onCompleted` handlers in the mutation. Don't bother changing now, we can improve this later.